src/Functions/CustomWeekTransforms.h

```diff
@@ -134,14 +134,17 @@ struct WeekTransformer
     void
     vector(const FromVectorType & vec_from, ToVectorType & vec_to, UInt8 week_mode, const DateLUTImpl & time_zone) const
     {
+        using ValueType = typename ToVectorType::value_type;
         size_t size = vec_from.size();
         vec_to.resize(size);
 
         for (size_t i = 0; i < size; ++i)
+        {
```

Comment:
Usually I'm using the same style as in the file, even though I prefer to use parentheses always.
However, this for loop has if/else, and even though it is work as expected, it is even more easier to make an error here:

So that's why I prefer:

```c
for (;;)
{
  if (foo)
    make_foo();
  else
    make_bar();
}
```

Over:

```c
for (;;)
  if (foo)
    make_foo();
  else
    make_bar();
```

Note, that I did not put extra parentheses for if/else.

But anyway, let me know what you thing, I'm to revert this change.

P.S. and if I did not have to change the lines inside if/else I wouldn't touch the whole hunk.

Reply:
It's okay and I see your argument.